app/controllers/OrchestratorController.scala

```diff
@@ -44,16 +44,12 @@ class OrchestratorController @Inject()(repository: FileNotificationRepositories,
               Future(BadRequest("Failed to parse to model"))
             },
             notifications => {
-              repository.storeFileNotifications(notifications).map {
-                response =>
-                  response.status match {
-                    case OK =>
-                      Ok("")
-                    case _ =>
-                      logger.error(s"[OrchestratorController][receiveSIDESNotifications] Mongo returned unknown status code: ${response.status} ")
-                      logger.debug(s"[OrchestratorController][receiveSIDESNotifications] Failure response body: ${response.body}")
-                      Status(response.status)
-                  }
+              mongoService.insertNotificationRecordsIntoMongo(notifications).map {
+                case true =>
+                  Ok("File Notification inserted")
+                case _ =>
```

Comment:
its not urgent to change but that there's no necessity to apply this matching as its a boolean

Reply:
Have updated the case to specify that the case is either true or false

Reply:
We have a ticket in the next Sprint looking at failures in saving to mongo so we can change then if needed
https://jira.tools.tax.service.gov.uk/browse/PRM-900